python-threatexchange/threatexchange/exchanges/clients/stopncii/api.py

```diff
@@ -44,6 +45,8 @@ class StopNCIICSPFeedbackValue(enum.Enum):
     """The feedback that a CSP has given on a hash"""
 
     Unknown = "Unknown"
+    # While undocumented, the API appears to use this value
+    Undefined = "Undefined"
```

Comment:
(blocking): Hey @Gnomeek in your comment you wrote

> Actually, I think it's better to add some fallback statements instead of adding enum value

However, it looks like the enum value is still here. Is that intentional? Your test is additionally asserting the existence.

Reply:
Both works for me. We can delete those two lines and change the test accordingly.